client/my-sites/plans/jetpack-plans/product-lightbox/included-product-list.tsx

```diff
@@ -18,21 +19,33 @@ const IncludedProductListItem: React.FC< IncludedProductListItemProps > = ( {
 } ) => {
 	const product = slugToSelectorProduct( productSlug );
 	return (
-		<div className="product-lightbox__included-product-list-item">
-			<div className="product-lightbox__included-product-list-item-icon">
-				<img alt="" src={ getProductIcon( { productSlug } ) } />
-			</div>
+		<>
+			<div className="product-lightbox__included-product-list-item">
+				<div className="product-lightbox__included-product-list-item-icon">
+					<img alt="" src={ getProductIcon( { productSlug } ) } />
+				</div>
 
-			<div className="product-lightbox__included-product-list-item-content">
-				<h1 className="product-lightbox__included-product-list-item-title">
-					{ product?.shortName }
```

Comment:
Can product be `null` or `undefined` here? If that is the case, what do you think about hiding the entire block instead of conditionally rendering the text?

Reply:
In terms of mapping (slugs to products), it's impossible to be null (because we have mapped all the products we use). But `slugToSelectorProduct` could possibly return null if it receives an invalid slug, that's why we need to handle that there.